tests/network-bridge-firewall

```diff
@@ -148,12 +148,46 @@ firewallTests() {
     lxc delete -f c1
 }
 
+networkLimitsPriorityTest() {
```

Comment:
Can we include nftables somewhere in the function name to show its nftables specific?

Reply:
Fixed.